community/kernel/src/test/java/org/neo4j/kernel/builtinprocs/StubKernelTransaction.java

```diff
@@ -73,6 +73,18 @@ public Read dataRead()
         throw new UnsupportedOperationException( "not implemented" );
     }
 
+    @Override
+    public Read stableDataRead()
+    {
+        return null;
```

Comment:
If we expect this to not be invoked, prefer throwing an exception.

Reply:
I agree, missed that here.